### ISSUE TITLE
Option to disable the "message delete" button

### ISSUE THREAD

**Issue by vagner-dias** [chatwoot/chatwoot#5950]
Hello, 

Could you please tell me if there is an option to disable the "message delete" button in chatwoot?

![image](https://user-images.githubusercontent.com/86367839/204010659-46893728-b467-4cb0-b71e-9946e84ad4ca.png)

**Comment by tejaswinichile** [comment]
@vagner-dias No; there is no functionality to disable the delete message. But we could surely include it in the future if it seems to be useful.

**Comment by amdKravtsov** [comment]
Operators should not have a delete message button.
Because the operator can be rude to the client, delete messages and the supervisor will not know about it.
Such a button can only be left with the supervisor to delete messages if the operator or client has sent personal data to the chat

**Comment by abdellani** [comment]
@tejaswinichile 
I take care of this.

**Comment by aris-infradigital** [comment]
I checked that the message is not deleted in the table but the content was modified into "This message was deleted". Maybe we can store also the previous/original message in another column or not modify it in DB.

**Comment by LiamAshdown** [comment]
Hi @sojan-official,

Would it be okay If I take this feature request on?

Also, for triaging shall I continue to mention you or not mention anyone.

Thanks.

**Comment by sojan-official** [comment]
@LiamAshdown  Please go ahead. 
It should be alright. Feel free to mention if you need any help while training 😄

**Comment by tafodinho** [comment]
Hi @sojan-official, 

I would love to know if this issue has been resolved already.
As it is something I will love to work on for my first PR.

Thanks

**Comment by leandrovip** [comment]
> I checked that the message is not deleted in the table but the content was modified into "This message was deleted". Maybe we can store also the previous/original message in another column or not modify it in DB.

+1

**Comment by shubh0284** [comment]
Hi, I would like to work on this issue. Can you please assign it to me?

**Comment by kevinjosh10** [comment]
Hey guys is this feature added or any help needed ? im looking for my first PR

**Comment by joao-baza-sup-uf** [comment]
> Hey guys is this feature added or any help needed ? im looking for my first PR

https://github.com/chatwoot/chatwoot/pull/14894

If you allow me to give a ideia, i think the delete button for channels must have one more step to confirm, because the cascade delete messages, medias, configs, so if someone delete wrongly, the damage is huge, @kevinjosh10
